zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/adapters/ElasticsearchAdapter.java

```diff
@@ -22,10 +22,13 @@ class ElasticsearchAdapter implements ClientAdapter {
   private final ElasticsearchClient client;
   private final ElasticsearchEngineClient searchEngineClient;
 
+  // TODO: private final ElasticsearchAsyncClient asyncClient;
+
   ElasticsearchAdapter(final ExporterConfiguration configuration) {
     final var connector = new ElasticsearchConnector(configuration.getConnect());
     client = connector.createClient();
     searchEngineClient = new ElasticsearchEngineClient(client);
+    // TODO: asyncClient = connector.createAsyncClient(); // for Archiver
```

Comment:
The search client we can use?

Reply:
That is just implementation details. If we want to wrap asyncClient with in the searchclient, we can do that. Not sure if that has any benefit.